src/standalone-js/src/deckgl.js

```diff
@@ -0,0 +1,178 @@
+/* global window, document */
+/* eslint-disable max-statements */
+import Mapbox from './mapbox';
+
+const CANVAS_STYLE = {
+  position: 'absolute',
+  left: 0,
+  top: 0,
+  width: '100%',
+  height: '100%'
+};
+
+/**
+ * @params container (Element) - DOM element to add deck.gl canvas to
+ * @params map (Object) - map API. Set to falsy to disable
+ * @params controller (Object) - Controller class. Leave empty for auto detection
+ */
+class DeckGL {
```

Comment:
A thought: Would it at all be possible to just inherit from `Deck` and add support for a few additional things (map, controllers)? 

It would certainly help to make me feel a little more comfortable with this PR. My thinking is that I want to keep the two JS efforts as tightly related as possible and make sure that as we go forward we implement and fix things once. Or well end up in complicated discussions where the two solutions have different issues and are pulling in different directions etc :)